lib/tasks/scripts/message_patients.rb

```diff
@@ -59,7 +60,19 @@ def send_messages
   end
 
   def contactable_patients
-    @contactable_patients ||= patients.contactable
+    @contactable_patients ||= if contactable
+      patients.contactable
+    else
+      patients
+    end
+  end
+
+  def phone_number_for(patient)
+    if contactable
+      patient.latest_mobile_number
+    else
+      patient.latest_phone_number
```

Comment:
What happens if the patient doesn't have a phone number? Is that a possible case?

Reply:
Great call. In that case we should most likely no-op the patient. I'll put that in 👍